### PR TITLE
chore: use file paths io url routes

### PR DESCRIPTION
**Pull request by stijnmoreels**
Use relative file paths instead of URL routes for more stable linking of sub-pages.

Related to https://github.com/arcus-azure/arcus/issues/191

### CODE REVIEW

File: docs/preview/02-Features/sinks/azure-application-insights.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ var host = Host.CreateDefaultBuilder()
 
                     string instrumentationKey = context.Configuration["ApplicationInsights:InstrumentationKey"];
 
-                    if (!String.IsNullOrWhiteSpace(instrumentationKey))
+                    if (!string.IsNullOrWhiteSpace(instrumentationKey))
```
Comment (stijnmoreels) [inline docs/preview/02-Features/sinks/azure-application-insights.md:156]: Changed this to lower-case, bc it's also the way we do it in Arcus, and I think it's also recommended by Microsoft.

File: docs/preview/03-Guidance/use-with-dotnet-and-functions.md [hunk 2]
```diff
@@ -54,16 +54,13 @@ namespace Arcus.Samples.AzureFunction
 
             builder.Services.AddLogging(loggingBuilder =>
             {
-                loggingBuilder.ClearProvidersExceptFunctionProviders();
```
Comment (stijnmoreels) [inline docs/preview/03-Guidance/use-with-dotnet-and-functions.md:57]: Since we deprecated this method, let's also remove it in our docs.

Comment (fgheysels) [reply docs/preview/03-Guidance/use-with-dotnet-and-functions.md:57]: Thought I did that, but must have forgotten it.

File: docs/preview/02-Features/sinks/azure-application-insights.md (hunk 1 above)
Comment (fgheysels) [reply docs/preview/02-Features/sinks/azure-application-insights.md:156]: Idk what is 'recommended'.  Anyway, there should be no difference as `String` refers to `System.String` and `string` is just an alias for it.
cfr `long` -> `System.Int64` , `int` -> `System.Int32` etc...

**Review by fgheysels — APPROVED** [review]

**Pull request merged**